docs/source/reference/api/opendataset.rst

```diff
@@ -0,0 +1,74 @@
+=====================
+tensorbay.opendataset
+=====================
+
+.. currentmodule:: tensorbay.opendataset
+
+.. autosummary::
+   :nosignatures:
+   :toctree: api_rst/
+
```

Comment:
So we still need to maintain this list if the opendataset list changes?

Reply:
Yes, autosummary supports `recursive` method, but it is not suitable for our situation.